conda/core/path_actions.py

```diff
@@ -1042,3 +1042,34 @@ def target_full_path(self):
     def __str__(self):
         return ('ExtractPackageAction<source_full_path=%r, target_full_path=%r>'
                 % (self.source_full_path, self.target_full_path))
+
+
+class DumpRecordAction(PathAction):
```

Comment:
I suppose I can clone the record object and remove the fields that we don't want to be stored in the json file. Is that what you have in mind ?